src/test/kotlin/com/exactpro/th2/common/utils/message/transport/MessageUtilsTest.kt

```diff
@@ -273,6 +275,45 @@ class MessageUtilsTest {
         assertNull(map.getFieldSoft("map-list", "0", "fake", "fake"))
     }
 
+    @Test
+    fun `contains filed`() {
```

Comment:
```suggestion
    fun `contains field`() {
```

I would also suggest making it a parameterized test. Because right now if any of those assertions fails others won't be checked